tests/files_test_activation_responses/activation_responses_examples/alara_params.txt

```diff
@@ -0,0 +1,21 @@
+material_lib alara_matlib
+element_lib data/nuclib
+data_library alaralib data/fendl2.0bin
+
+#     flux name    fluxin file   norm   shift   unused
+flux  my_flux     alara_fluxin  1e10     0      default
```

Comment:
I don't think `1.0` is a safer number here. This `1e10` here is easier for users to notice that this is a number need to be replaced! If it is `1.0`, users may ignore it and not change it.

Reply:
Sure. But by  default a user of MCNP might produce a normalized tally and provide it to ALARA without noticing the normalization that has to change.

I know that it depends, but most probably a mesh tally would be produced with a provided normalization MCNP. Plus, and error of not normalizing inside ALARA parameters file will result in a termination of the run because of truncation while a normalization applied twice might go for longer!.

Reply:
It's OK to do as what you mentioned, if there is only one multiplier (i.e., the power of the plant is a constant).

But how about the case that the power of plant changes at different irradiation time? Eg. `P1` MW in the first year, and `P2` MW in the second year. Users must be noticed to change the normalization when they use ALARA.

Reply:
Sure. I didn't think about that!. Maybe add a comment on the config file where the meshtal file is to be specified that the user has to change normalization in the alara_parameters file.

Reply:
Added a comment in https://github.com/pyne/pyne/pull/1123/commits/c57c66eb4fbe1425f0f19b15cb48bb78f30d6790.